### PR TITLE
refactor(console): hide navigation anchors from guide in compact view

### PR DESCRIPTION
**Pull request by charIeszhao**
## Summary

Support compact view in Guide `<Steps>` component. Hide those navigation anchors in compact view (such as in a drawer)

## Testing

Locally tested

## Checklist

- [x] This PR is not applicable for the checklist

### CODE REVIEW

File: packages/console/src/assets/docs/guides/index.ts [hunk 1]
```diff
@@ -86,12 +85,7 @@ const guides: Readonly<Guide[]> = Object.freeze([
     Component: lazy(async () => import('./spa-vue/README.mdx')),
     metadata: spaVue,
   },
-  {
-    id: 'web-chatgpt',
-    Logo: lazy(async () => import('./web-chatgpt/logo.svg')),
-    Component: lazy(async () => import('./web-chatgpt/README.mdx')),
-    metadata: webChatgpt,
-  },
+
```
Comment (gao-sun) [inline packages/console/src/assets/docs/guides/index.ts:88]:
```suggestion
```

File: packages/console/src/mdx-components-v2/Steps/index.tsx [hunk 2]
```diff
@@ -29,7 +30,7 @@ const findScrollableElement = (element: Nullable<HTMLElement>): Nullable<HTMLEle
   return findScrollableElement(element.parentElement);
 };
 
-export default function Steps({ children: reactChildren }: Props) {
+export default function Steps({ hasNavigationAnchors = true, children: reactChildren }: Props) {
```
Comment (gao-sun) [inline packages/console/src/mdx-components-v2/Steps/index.tsx:33]: no need to pass prop here. you can use `GuideContext`

Comment (charIeszhao) [reply packages/console/src/mdx-components-v2/Steps/index.tsx:33]: Ah, good point

File: packages/console/src/assets/docs/guides/index.ts (hunk 1 above)
Comment (charIeszhao) [reply packages/console/src/assets/docs/guides/index.ts]: Updated

**Review by gao-sun — APPROVED** [review]

**Review by wangsijie — APPROVED** [review]

**Pull request merged**